src/main/java/bio/terra/common/iam/AuthenticatedUserRequest.java

```diff
@@ -40,6 +48,58 @@ public String getToken() {
     return token;
   }
 
+  public FlightMap putIn(FlightMap flightMap, String key) {
+    return putMessage(flightMap, key, toModel());
+  }
+
+  public static AuthenticatedUserRequest getFrom(FlightMap flightMap, String key) {
+    AuthenticatedUserRequestModel model =
+        getMessage(flightMap, key, AuthenticatedUserRequestModel.parser());
+    return fromModel(model);
+  }
+
+  /**
+   * This could be put on FlightMap as a part of Stairway. Broken out to a separate function here to
+   * illustrate.
+   */
+  @VisibleForTesting
+  static FlightMap putMessage(FlightMap flightMap, String key, Message message) {
```

Comment:
Worth to add explain how 
https://github.com/DataBiosphere/stairway/blob/develop/src/main/java/bio/terra/stairway/FlightMap.java#L33
 and https://github.com/DataBiosphere/stairway/blob/develop/src/main/java/bio/terra/stairway/FlightMap.java#L51
would work in this case?

Reply:
As those are package private, I can't demo them in this PR :(
Theoretically, they work the same as they do for any String value stored in a FlightMap. As-is in the demo, we're storing the binary serialized proto wire format as a String; it doesn't matter to Stairway that the String has special meaning.